recipe/meta.yaml

```diff
@@ -23,13 +23,17 @@ requirements:
   run:
     - python
     - absl-py >=0.1.6
-    - enum34 >=1.1.6  # [py<34]
-    - mock >=2.0.0    # [py27]
-    - numpy >=1.12.1
-    - protobuf >=3.4.0
+    - astor >=0.6.0
+    - gast >=0.2.0
+    - numpy >=1.13.3
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
-    - backports.weakref >=1.0rc1
+    - protobuf >=3.4.0
+    - tensorboard >=1.8.0, <1.9.0
+    - termcolor >=1.1.0
+    - grpcio >=1.8.6
+    - mock >=2.0.0                # [py27]
+    - enum34 >=1.1.6              # [py<34]
+    - backports.weakref >=1.0rc1  # [py<34]
```

Comment:
Should be `- backports.weakref  # [py2k]` I think

Reply:
In the setup.py it is coded as:
```
# weakref.finalize and enum were introduced in Python 3.4
if sys.version_info < (3, 4):
  REQUIRED_PACKAGES.append('backports.weakref >= 1.0rc1')
  REQUIRED_PACKAGES.append('enum34 >= 1.1.6')
```
It is still better to use `# [py2k]`?

Reply:
They have exactly the same effect on conda-forge, doesn't matter.